src/Groups/matrices/form_group.jl

```diff
@@ -546,6 +546,14 @@ function orthogonal_sign(G::MatrixGroup)
                                   codomain(iso_oscar_gap(R)))
     sign = GAP.Globals.MTX.OrthogonalSign(M)
     sign == GAP.Globals.fail && return nothing
+    # If the characteristic is odd and there is an invariant
+    # antisymmetric bilinear form then `GAP.Globals.MTX.OrthogonalSign`
+    # does *not* return `fail`,
+    # see https://github.com/gap-system/gap/issues/4936.
+    if isodd(characteristic(R))
+      Q = GAP.Globals.MTX.InvariantQuadraticForm(M)
+      Q == - GAP.Globals.TransposedMat(Q) && return nothing
```

Comment:
If we want to change GAP to return `fail` in this case in the future, perhaps we should defensively code this like this perhaps:
```suggestion
      if Q === GAP.Globals.fail ||Q == - GAP.Globals.TransposedMat(Q)
        return nothing
```

Reply:
O.k.
(With a change of `GAP.Globals.MTX.InvariantQuadraticForm`, the behaviour of `GAP.Globals.MTX.OrthogonalSign` will change such that this function will return `GAP.Globals.fail` if there is no nondegenerate invariant quadratic form, i.e., the first check for `GAP.Globals.fail` will be successful in this case.)